internal/server/subscriptionlistener.go

```diff
@@ -155,8 +155,15 @@ func (s *Server) handleDataChange(dcn *ua.DataChangeNotification) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
+	var data any
```

Comment:
Would it make sense to move this inside the for loop? Once it leaves the scope of the for loop, it can't be reused.